podman/domain/images_manager.py

```diff
@@ -200,8 +202,8 @@ def prune(
         response = self.client.post("/images/prune", params=params)
         response.raise_for_status()
 
-        deleted: List[Dict[str, str]] = []
-        error: List[str] = []
+        deleted: list[dict[str, str]] = []
+        error: list[str] = []
```

Comment:
```suggestion
        deleted: builtins.list[dict[str, str]] = []
        error: builtins.list[str] = []
```
May be more correct. I did not research where in the scope this list() is anchored.

Reply:
I changed all the ambiguous for the linter, probably, for the human eye is better to be more explicit with the checks as you suggest. I'll change it this way